assets/scripts/index.js

```diff
@@ -0,0 +1,126 @@
+$(document).ready(function(){
+
+  document.querySelector(".js-nav-toggle").addEventListener("click",toggleMenuFunction);
+  document.querySelector(".js-top").addEventListener("click",toTop);
+  window.onscroll = function() {scrollFunction()};
+
+  function toggleMenuFunction() {
+    let burger = document.querySelector(".nav__menu-toggle");
+    let burgerMenu = document.querySelector(".nav__menu");
+    let mainnav = document.querySelector(".nav");
```

Comment:
@edwartexe to avoid confusion with variable names you should use a case rule. As for javascript we use `camelCase`
```suggestion
    let mainNav = document.querySelector(".js-nav");
```